migrator/migrations/m_203_to_m_204_openshift_policy_exclusions_for_4_5/migration.go

```diff
@@ -0,0 +1,157 @@
+// Code generated by make bootstrap_migration generator. DO NOT EDIT.
+package m203tom204
+
+import (
+	"context"
+	"embed"
+
+	"github.com/stackrox/rox/generated/storage"
+	"github.com/stackrox/rox/migrator/migrations"
+	"github.com/stackrox/rox/migrator/types"
+	"github.com/pkg/errors"
+	"github.com/stackrox/rox/migrator/migrations/m_203_to_m_204_openshift_policy_exclusions_for_4_5/schema"
+	"github.com/stackrox/rox/migrator/migrations/policymigrationhelper"
+	"gorm.io/gorm"
+)
+
+const (
+	startSeqNum = 203
+)
+
+var (
+	migration = types.Migration{
+		StartingSeqNum: startSeqNum,
+		VersionAfter:   &storage.Version{SeqNum: int32(startSeqNum + 1)},
+		Run: func(databases *types.Databases) error {
+			   err := updatePolicies(databases.GormDB)
+		      if err != nil {
+		      return errors.Wrap(err, "updating policies")
+	    }
+		return nil
+	},
+	}
+	//go:embed policies_before_and_after
+	policyDiffFS embed.FS
+
+	// We want to migrate only if the existing policy sections,name and description haven't changed.
+	fieldsToCompare = []policymigrationhelper.FieldComparator{
+		policymigrationhelper.DescriptionComparator,
+		policymigrationhelper.PolicySectionComparator,
+		policymigrationhelper.NameComparator,
+	}
+
+	policyDiffs = []policymigrationhelper.PolicyDiff{
```

Comment:
nit:  order these alphabetically by filename to make it easier to compare to the file listings.